src/@next/Banner/Banner.tsx

```diff
@@ -9,9 +9,10 @@ import {
   StyledBannerTitle,
   StyledBannerTitleContainer,
   StyledCloseIconWrapper,
+  StyledFixedCloseIconWrapper,
 } from './BannerStyle';
 
-export type BannerProps = {
+export interface BannerProps extends React.HTMLAttributes<HTMLDivElement> {
```

Comment:
Before, if we passed an extra prop (e.g., style), it caused a typescript error. This is unrelated to my functionality, but I think needed to prevent getting those unnecessary typescript warnings.